parlai/utils/conversations.py

```diff
@@ -93,6 +95,74 @@ def save_metadata(
             f.write(json.dumps(metadata))
 
 
+class Turn(AttrDict):
+    """
+    Utility class for a dialog turn.
+    """
+
+    def __init__(self, id=None, text=None, **kwargs):
+        super().__init__(self, id=id, text=text, **kwargs)
+
+
+class Conversation:
+    """
+    Utility class for iterating through a single episode.
+
+    Used in the context of the Conversations class.
+    """
+
+    def __init__(self, episode):
```

Comment:
so the `Conversation` assumes that an episode is formatted a certain way right? would it be good to document here what fields the `Conversation` class is expecting (since you can't just load with any arbitrarily built episode)?

Reply:
Yeah it assumes the episode is formatted in the way returned by the World Logger. Aka it is a list of act pairs, which are in turn a list of acts (aka Messages/dicts). Will document